hws/src/dr_remote.py

```diff
@@ -0,0 +1,113 @@
+#SPDX-License-Identifier: BSD-3-Clause
+#Copyright (c) 2024 NVIDIA CORPORATION. All rights reserved.
+
+import importlib
+import time
+import sys
+import os
+
+from src.dr_common import *
+from src.dr_db import _config_args
+
+SSH_OPTIONS = "-q -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -oPubkeyAuthentication=no"
+
+def remote_cmd(cmd, host, user, password, time_out=60):
+    if _config_args.get("remote_dep_lib") == False:
+        globals()["pexpect"] = importlib.import_module("pexpect")
+        globals()["tempfile"] = importlib.import_module("tempfile")
+        _config_args["remote_dep_lib"] = True
+
+    f = tempfile.mktemp()
+    fout = open(f, 'w')
+
+    cmd = "ssh %s@%s %s \"%s\"" % (user, host, SSH_OPTIONS, cmd)
+    p = pexpect.spawnu(cmd, timeout=time_out)
+  
+    i = p.expect(['[Yy]es', 'YES', '[Pp]assword: '])
+    if i < 2:
+        p.sendline('YES')
+        i = p.expect(['[Yy]es', 'YES', '[Pp]assword: '])
+        if i < 2:
+            p.sendline('Y')
+            i = p.expect(['[Yy]es', 'YES', '[Pp]assword: '])
+            if i < 2:
+                print("Connection error")
+                sys.exit(1)
+
```

Comment:
Can you put here a two lines of comment that describes what kind of output you expect?
What are the two expected responses from the remote machine?

Reply:
if machine expects YES or Y, else will see the error and add another expected output